### PR TITLE
Update  git / workflow / finding-commits /

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Change Link Name

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/git/workflow/finding-commits/search-by-commit-message-keyword.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/git/workflow/finding-commits/search-by-commit-message-keyword.md?ref=473ed67833a3c84746cd259517f36101ae965e0b)

File: git/workflow/finding-commits/search-by-commit-message-keyword.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](http://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){website}'
+  - '[Viewing The Commit History Official Documentation](http://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){website}'
```
Comment (kapnobatai137) [inline git/workflow/finding-commits/search-by-commit-message-keyword.md]:
```suggestion
  - '[Viewing The Commit History](http://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){documentation}'
```

File: git/workflow/finding-commits/search-for-commits-by-author.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](http://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){website}'
+  - '[Viewing The Commit History Official Documentation](http://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){website}'
```
Comment (kapnobatai137) [inline git/workflow/finding-commits/search-for-commits-by-author.md]:
```suggestion
  - '[Viewing The Commit History](http://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){documentation}'
```

File: git/workflow/finding-commits/specify-a-range-of-commits-using-double-dot-syntax.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](http://git-scm.com/book/en/v2/Git-Tools-Revision-Selection){website}'
+  - '[Double Dot Syntax Official Documentation](http://git-scm.com/book/en/v2/Git-Tools-Revision-Selection){website}'
```
Comment (kapnobatai137) [inline git/workflow/finding-commits/specify-a-range-of-commits-using-double-dot-syntax.md]:
```suggestion
  - '[Double Dot Syntax](http://git-scm.com/book/en/v2/Git-Tools-Revision-Selection){documentation}'
```

**Pull request merged**